docs/css/baseline-align.css

```diff
@@ -138,4 +138,19 @@ header {
 
 .bypass-block-target:focus {
 	outline: none;
-}
\ No newline at end of file
+}
+
+code {
+    color: #C00000 !important;
+}
+
+.code {
+    background-color: #bbb;
+    background-color: rgba(222,222,222,.5);
```

Comment:
Did you intend to define two different colors for background-color?

Reply:
Good eye. Will update to remove redundant #bbb.

Reply:
Done.